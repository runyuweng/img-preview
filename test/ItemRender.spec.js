import React from 'react';
import { mount } from 'enzyme';
import ImgItem from '../lib/item';

const imgUrl = 'https://avatars2.githubusercontent.com/u/16834265?s=400&u=dd37eb4c227696e1fe47e8354d59b5db53be6959&v=4'

describe('ImgItem', () => {

  const createImgITem = (props) => (
    <ImgItem
      src={props.imgUrl}
    />
  );

  it('Item renders correctly', () => {
    const wrapper = mount(createImgITem({ imgUrl })); 
    expect(wrapper.render()).toMatchSnapshot();
  });
});
