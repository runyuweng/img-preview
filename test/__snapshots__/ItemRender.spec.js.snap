// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImgItem Item renders correctly 1`] = `
<div
  class="thumbnail-container"
>
  <div
    class="thumbnail"
  >
    <div
      class="loading"
    >
      <div
        class="loading-container"
      >
        <span />
        <span />
        <span />
        <span />
      </div>
      <img
        src="https://avatars2.githubusercontent.com/u/16834265?s=400&u=dd37eb4c227696e1fe47e8354d59b5db53be6959&v=4"
      />
    </div>
  </div>
</div>
`;
